src/components/form/InputPath.js

```diff
@@ -16,12 +16,12 @@ export default class InputFilename extends Component {
     if (type == 'posts') {
       const date = moment().format('YYYY-MM-DD');
       placeholder = `${date}-your-title.md`;
-    }else if (type == 'datafiles') {
-      placeholder = 'your-filename.yml';
+    }else if (type == 'data files') {
+      placeholder = '<directory/>your-filename.yml';
```

Comment:
Ah! I see. I'll settle for the your suggestion then `<directory> / your-filename.yml`